### PR TITLE
Small things

### PR DESCRIPTION
**Pull request by necessarily-equal**

### CODE REVIEW

**Comment by necessarily-equal** [comment]
Note I've updated the const fixes commit and sneaked in two other commits. This should be ready for review.

**Comment by necessarily-equal** [comment]
Also, I've tested this with a the released 0.52 assets+gamelogic, and it works, but I must say I'm a bit wary as this could be an ABI break.

File: src/engine/qcommon/q_math.cpp [hunk 1]
```diff
@@ -200,7 +200,7 @@ int DirToByte( vec3_t dir )
 	bestd = 0;
 	best = 0;
 
-	for ( i = 0; i < NUMVERTEXNORMALS; i++ )
+	for ( size_t i = 0; i < NUMVERTEXNORMALS; i++ )
```
Comment (slipher) [inline src/engine/qcommon/q_math.cpp:203]: change back to int?

File: src/engine/qcommon/q_math.cpp [hunk 2]
```diff
@@ -559,6 +559,18 @@ void ExponentialFade( float *value, float target, float lambda, float timedelta
...
 }
 
+/*
+=============
+ExponentialFade
+
+Fades one value towards another exponentially
+=============
+/
+inline float EaseInCubic(float percentage, float from, float to)
```
Comment (slipher) [inline src/engine/qcommon/q_math.cpp:585]: what's this

Comment (necessarily-equal) [reply src/engine/qcommon/q_math.cpp]: another thing not meant to be commited :eyes:

**Pull request merged**